### ISSUE TITLE
[BUG] asyncApi document v3.0.0 doesn't get rendered

### ISSUE THREAD

**Issue by kingofdisasterr** [asyncapi/jasyncapi-idea-plugin#105]
### Describe the bug.

given document:
```yml
asyncapi: 3.0.0
info:
  version: '1'
  title: 'my fancy ansyncapi'
channels:
  the.channel:
    description: 'a fancy channel'
```

shows a error in the renderer.

AsyncAPI intellij plugin version: 2.6.0+jre17

### Expected behavior

the document gets rendered.

### Screenshots

![image](https://github.com/user-attachments/assets/16fb21dd-356e-4f4c-a6bb-56ef387013d6)

### How to Reproduce

1. the document renders when i put in `2.5.0` as version
2. change the version to `3.0.0` and the error shows up

### 🥦 Browser

None

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

**Comment by Pakisan** [comment]
You are using preview panel from folks at JetBrains. Close it and open built-in web preview

**Comment by kingofdisasterr** [comment]
when i try to render with the web preview (i.e. chrome) it leads me to a '404 Not Found' error:

<img width="617" alt="image" src="https://github.com/user-attachments/assets/296baffe-1189-4484-9b8c-1978a22d659b">

**Comment by Pakisan** [comment]
Yep, it's known issue with temp file resolving 

I have already resolved it in upcoming release 2.7.0, by rendering content directly instead of using temporal files

I can drop you `jar`, to install new version manually and provide feedback
